### DISCUSSION TITLE
filament importer excel v 3.1

### DISCUSSION THREAD

**Discussion by nimahamidpour1994** [filamentphp/filament#10520] [Help]
hi
 i user import action in latest filament feature release
how can  when i upload excel when some row  error  validation download csv that get error rows data ? i cant find any thing about
and how can use getCompletedNotificationBody method to tell user some rows has error

**Comment by lukeholder** [comment]
Yeah I also cant see anything in the docs on the expected process for downloading the csv containing rejected rows? I wonder if they expect us to make a `FailedImportRowsResource`?

**Reply by lukeholder** [reply]
OK figured it out. They go to into notifications! Make sure you have 

```
->databaseNotifications()
->databaseNotificationsPolling('30s')
```

set up on your panel or notifications will not show up.

@nimahamidpour1994
